extension/chrome/elements/compose-modules/compose-storage-module.ts

```diff
@@ -147,45 +173,43 @@ export class ComposeStorageModule extends ViewModule<ComposeView> {
         if (updates.length) {
           await Promise.all(updates.map(async (update) => await ContactStore.update(undefined, email, update)));
         }
-        const [preferred] = await ContactStore.get(undefined, [email]);
-        return preferred ?? PUBKEY_LOOKUP_RESULT_FAIL;
-      } else {
-        return PUBKEY_LOOKUP_RESULT_FAIL;
+        const emailWithKeys = await ContactStore.getOneWithAllPubkeys(undefined, email);
```

Comment:
Actually, this method is re-reading the storage to provide the "result". I think it should simply return 'fail' | 'success' or boolean

Reply:
Still don't catch how return of public key can be changed to just boolean?